test/closure/zone_externs_failed.js

```diff
@@ -0,0 +1,445 @@
+/**
+* @license
+* Copyright Google Inc. All Rights Reserved.
+*
+* Use of this source code is governed by an MIT-style license that can be
+* found in the LICENSE file at https://angular.io/license
+*/
+
+/**
+ * @fileoverview Externs for zone.js
```

Comment:
this file also looks like a lot of maintenance work. What is different between this and the other one? Could we get the same test coverage by just calling closure with no externs at all?